test/migration_test.rb

```diff
@@ -230,4 +231,18 @@ def test_status_without_pending_migrations
     end
 
   end
+
+  class NameSpaceTestCase < TestCase
+    def setup
+      Mongoid::Migrator.namespace = 'db:mongoid'
+      super
+      Mongoid::Migrator.migrations_path = ["db/mongoid"]
+    end
+
+    def test_migrations_path
+      assert_equal ["db/mongoid"], Mongoid::Migrator.migrations_path
+      Mongoid::Migrator.migrations_path += ["engines/my_engine/db/migrate"]
+      assert_equal ["db/mongoid", "engines/my_engine/db/migrate"], Mongoid::Migrator.migrations_path
+    end
```

Comment:
This is not really testing the namespacing of tasks ^^

Reply:
It doesn't but practically speaking if one really wants to de-conflict with `ActiveRecord` they should be modifying both the `namespace` and the `path`. Coz if the actual migration files of AR and Mongo live in the same path there rake tasks will still fail. Hence I tested both in the same test case.

Speaking it out loud, this caveat should be mentioned in the README file too

Reply:
Sure, no problem with that, what I mean is that the namespacing itself is not tested, if you remove all the changes in `database.rake` this spec still pass isn't it? I'm not sure if it's easy to do so but I think we should verify the list of rake tasks generated, that their name is correct.